drkafka/scripts/run.sh

```diff
@@ -2,9 +2,6 @@
 
 set -x
 
-
 java8 -cp lib/*:kafkaoperator-0.2.3.jar  -Dlog4j.configurationFile=file:./log4j2.xml  \
-      com.pinterest.doctorkafka.DoctorKafkaMain   \
-      -topic brokerstats -zookeeper datazk001:2181/data07  \
```

Comment:
remove these parameters will prevent the script from running properly .  can we just change `datazk001` to a generic host name?

Reply:
Due to migration to dropwizard we are changing the way Dr. Kafka will launch.

Reply:
I tested with 

     java8  -cp lib/*:./doctorkafka-0.2.3.jar  -Dlog4j.configurationFile=file:./config/log4j2.xml  \
        com.pinterest.doctorkafka.DoctorKafkaMain -config doctorkafka.prod.properties  

DoctorKafka could not be started properly.  The following is the output:

    08:10:05.128 [main] INFO  org.hibernate.validator.internal.util.Version - HV000001: Hibernate Validator 5.4.2.Final
    unrecognized arguments: '-config'
    usage: java -jar doctorkafka-0.2.3.jar
           [-h] [-v] {server,check} ...

    positional arguments:
      {server,check}         available commands

    named arguments:
      -h, --help             show this help message and exit
      -v, --version          show the application version and exit

Reply:
@yuyang08 the correct way to launch the application is listed in the details section.

Reply:
It would be very helpful to users to track configuration in a file instead of System properties. We can do a second iteration on this and merge these two configuration types in the future.

Another alternative would be to retrofit System property configurations in dropwizard however this may be counter intuitive from a Dropwizard pattern.